packages/react/src/dialog-content/DialogContent.tsx

```diff
@@ -50,22 +40,7 @@ export const DialogContent = forwardRef<HTMLDivElement, DialogContentProps>(
                 {...styles.content({ size }, className)}
                 {...props}
               >
-                <RadixDialog.Content ref={ref}>
-                  {children}
-
-                  {withCloseButton && (
-                    <Button
-                      appearance="subtle"
-                      aria-label="Close"
-                      asChild
-                      icon={<IconX />}
-                      size="sm"
-                      {...styles.close()}
-                    >
-                      <RadixDialog.Close />
-                    </Button>
-                  )}
-                </RadixDialog.Content>
+                <RadixDialog.Content ref={ref}>{children}</RadixDialog.Content>
```

Comment:
Vai, when there's no focusable item in the content, opening the drawer has a `black` outline for edge / `blue` outline for chrome.